contracts/governance/Vesting/VestingRegistryLogic.sol

```diff
@@ -26,6 +26,7 @@ contract VestingRegistryLogic is VestingRegistryStorage {
         uint256 vestingCreationType
     );
     event TokensStaked(address indexed vesting, uint256 amount);
+    event VestingDetailPushed(address indexed vesting, Vesting vestingDetail);
```

Comment:
```suggestion  
event VestingDetailsSet(address indexed vesting, Vesting vestingDetail);
```

Reply:
updated in this [commit](https://github.com/DistributedCollective/Sovryn-smart-contracts/pull/447/commits/79f72c867c17cd1f12ed1e726ee79e2c308a8e9e)